### PR TITLE
Add variant annotation process

### PR DESCRIPTION
**Pull request by marcellevstek**
* [x] Update CHANGELOG.rst for each commit separately:
  * Pay attention to write entries under the "Unreleased" section.
  * Mark all breaking changes as "**BACKWARD INCOMPATIBLE:**" and put them
    before non-breaking changes.
  * If a commit modifies a feature listed under "Unreleased" section,
    it might be sufficient to modify the existing CHANGELOG entry from previous
    commit(s).
* [x] Bump the process version:
  * **MAJOR version (first number)**: Backward incompatible changes (changes
    that break the api/interface). Examples: renaming the input/output, adding
    mandatory input, removing input/output...
  * **MINOR version (middle number)**: add functionality or changes in a
    backwards-compatible manner. Examples: add output field, add non-mandatory
    input parameter, use a different tool that produces same results...
  * **PATCH version (last number)**: changes/bug fixes that do not affect
    the api/interface. Examples: typo fix, change/add warning messages...
* [x] All inputs are used in process.
* [x] All output fields have a value assigned to them.

### CODE REVIEW

File: resolwe_bio/tests/processes/test_annotation.py [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+                },
+            )
+            clinvar_2 = self.run_process(
+                "upload-variants-vcf",
+                {
+                    "src": input_folder / "clinvar_2.vcf.gz",
+                    "species": "Homo sapiens",
+                    "build": "GRCh38",
+                },
+            )
```
Comment (jkokosar) [inline resolwe_bio/tests/processes/test_annotation.py:66]: what is the purpose, what does this test?

File: resolwe_bio/tests/processes/test_annotation.py [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+        self.assertDictEqual(
+            VariantAnnotationSerializer(variant[0].annotation).data["transcripts"][0],
+            expected_annotation["transcripts"][0],
+        )
+        self.assertFile(
+            variant_annotation, "vcf", output_folder / "annotated_variants.vcf.gz"
+        )
+
+        variant_annotation = self.run_process(
```
Comment (jkokosar) [inline resolwe_bio/tests/processes/test_annotation.py:147]: Should be followed by an `assert`.

**Review by jkokosar — COMMENTED** [review]
Please see the initial review. Looks great already.

File: resolwe_bio/tests/processes/test_annotation.py (hunk 1 above)
Comment (marcellevstek) [reply resolwe_bio/tests/processes/test_annotation.py:66]: It tries to update the annotations of existing variant.

File: resolwe_bio/tests/processes/test_annotation.py (hunk 2 above)
Comment (marcellevstek) [reply resolwe_bio/tests/processes/test_annotation.py:147]: Will add once we resolve the errors on the backend.

**Comment by jkokosar** [comment]
Please rebase

**Review by gregorjerse — APPROVED** [review]

**Review by jkokosar — APPROVED** [review]
Thank you for taking the time to write clean and efficient code! Please rebase.

**Pull request merged**